app/screens/Explore.js

```diff
@@ -751,14 +750,6 @@ const mapStateToProps = (state) => {
         })
       }
     })
-
-    nodes = {
```

Comment:
I think we still want to show currentWayEdit nodes also on the map. Could we add these into a different geojson source? And style that differently?